src/modules/Dashboard/hooks/useDecryptedActivityData.ts

```diff
@@ -85,6 +87,22 @@ export const useDecryptedActivityData = (
       }
     }
 
+    /*
+    Here we use 'any' because of differences in API responses when Export Data
+    and in Datawiz.
+    Thus, in dataviz summary the responses are retrived without details about:
+    'version', 'client.appId', 'client.appVersion'
+    */
+    logDataInDebugMode({
+      [`applet_v.${(rest as any as ExportAnswer)?.version ?? ''}/appId_${
```

Comment:
Can we use `rest as unknown as ExportAnswer`?

Reply:
agree